config/40-siteinfo.ini

```diff
@@ -3,79 +3,74 @@
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 resource = UNAVAILABLE
 
 
-; The resource_group setting should be set to the same value as used in the OIM 
-; registration at the goc 
+; Set "resource_group" to the name of the resource that you have registered
+; in the OSG topology repository at https://github.com/opensciencegrid/topology
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 resource_group = UNAVAILABLE
 
-; The sponsor setting should list the sponsors for your cluster, if your cluster
-; has multiple sponsors, you can separate them using commas or specify the 
-; percentage using the following format 'osg, atlas, cms' or 
-; 'osg:10, atlas:45, cms:45'  
+; Set "sponsor" to the sponsor(s) for your cluster.
+; If your cluster has multiple sponsors, separate them with commas
+; and specify the percentages using this format: "osg:10, atlas:45, cms:45".
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 sponsor = UNAVAILABLE
 
-; The site_policy setting should give an url that lists your site's usage
-; policy 
+; Set "site_policy" to a URL that lists your site's usage policy (if there
+; is such a URL).
 site_policy = UNAVAILABLE
 
-; The contact setting should give the name of the admin/technical contact
-; for the cluster
+; Set "contact" to the name of the admin/technical contact for the cluster.
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 contact = UNAVAILABLE
 
-; The email setting should give the email address for the technical contact
-; for the cluster 
+; Set "email" to the email address of the admin/technical contact for the cluster.
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 email = UNAVAILABLE
 
-; The city setting should give the city that the cluster is located in
+; Set "city" to the city that the cluster is located in.
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 city = UNAVAILABLE
 
-; The country setting should give the country that the cluster is located in
+; Set "country" to the country that the cluster is located in.
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 country = UNAVAILABLE
 
-; The longitude setting should give the longitude for the cluster's location
-; if you are in the US, this should be negative
-; accepted values are between -180 and 180
+; Set "longitude" to the longitude of the cluster's location.
+; Accepted values are between -180 (west) and 180 (east).
+; If you are in the US, this will be negative.
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 longitude = UNAVAILABLE 
 
-; The latitude setting should give the latitude for the cluster's location
-; accepted values are between -90 and 90
+; Set "longitude" to the latitude of the cluster's location.
+; Accepted values are between -90 (north) and 90 (south).
+; If you are in the US, this will be negative.
 ; 
 ; YOU WILL NEED TO CHANGE THIS
 latitude = UNAVAILABLE
 
-; This option has been deprecated and should not be used.  It has been replaced 
-; by the resource and resource_group settings
+; Do not use "site_name".  It has been replaced by "resource" and "resource_group".
```

Comment:
Why do we keep this around if it's deprecated?

Reply:
Dropping "site_name" entirely is in the todo list for the OSG 3.5 version (https://opensciencegrid.atlassian.net/browse/SOFTWARE-3125); it's considered a disruptive change because it would break existing configs. Until then I'd like to keep this comment as a reminder.